src/Router.js

```diff
@@ -6,63 +6,105 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
-import React, { Component, NavigationExperimental } from 'react-native';
-const {
-    AnimatedView: NavigationAnimatedView,
-    Card: NavigationCard,
-    RootContainer: NavigationRootContainer,
-    Header: NavigationHeader,
-    } = NavigationExperimental;
+import React, {
+  Component,
+  NavigationExperimental,
+  PropTypes,
+} from 'react-native';
+
 import Actions from './Actions';
 import getInitialState from './State';
 import Reducer from './Reducer';
 import DefaultRenderer from './DefaultRenderer';
 import Scene from './Scene';
 
-export default class extends Component {
+const {
+  RootContainer: NavigationRootContainer,
+} = NavigationExperimental;
+
+const propTypes = {
+  dispatch: PropTypes.func,
+};
+
+export default class Router extends Component {
```

Comment:
Since `propTypes` is assigned after the class declaration, `export default` should be moved to the end of the file.

```javascript
Router.propTypes = propTypes;

export default Router;
```